README.md

```diff
@@ -1196,6 +1196,39 @@ peered with.
 
 `peer_vpc` defines the identifier or name of the remote VPC.
 
+### Datasource Service Component
+```hcl-terraform
+data "aiven_service_component" "sc1" {
+    project = aiven_kafka.project1.project
+    service_name = aiven_kafka.service1.service_name
+    component = "kafka"
+    route = "dynamic"
```

Comment:
This should also define `kafka_authentication_method = "certificate"` as otherwise with current logic this would not match to anything.